Splash/Splash/Details/DetailsViewController.m

```diff
@@ -0,0 +1,166 @@
+//
+//  DetailsViewController.m
+//  Splash
+//
+//  Created by Shreya Vinjamuri on 7/18/22.
+//
+
+#import "DetailsViewController.h"
+#import "Parse/Parse.h"
+#import "PFImageView.h"
+#import "DetailsCell.h"
+#import "TimeViewController.h"
+#import "StatsCell.h"
+#import "Helper.h"
+
+@interface DetailsViewController () <UITableViewDelegate, UITableViewDataSource>
+@property (strong, nonatomic) IBOutlet UILabel *usernameLabel;
+@property (strong, nonatomic) IBOutlet PFImageView *profilePic;
+@property (strong, nonatomic) NSArray *scoreNames;
+@property (strong, nonatomic) IBOutlet UITableView *detailsTableView;
+@property (strong, nonatomic) IBOutlet UIButton *removeFriendButton;
+@property (strong, nonatomic) IBOutlet UIButton *addFriendButton;
+@property (strong, nonatomic) IBOutlet UIButton *challengeButton;
+@property (strong, nonatomic) PFUser *current;
+@property (strong, nonatomic) IBOutlet UIButton *onLBButton;
+@property (strong, nonatomic) NSArray *friends;
+@end
+
+@implementation DetailsViewController
+NSString *DetailsHeaderViewIdentifier = @"DetailsTableViewHeaderView";
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    self.usernameLabel.text = self.user[@"username"];
```

Comment:
I think we can break this up further.
For eg: 
move detailsTableView inside setupTableView()
move button, labels logic inside setupView()...